README.md

```diff
@@ -171,3 +171,39 @@ PicUpSDK.shared.enableService()
 ```swift
 PicUpSDK.shared.clearData()
 ```
+
+## Silent Push Notifications
+
+To receive updates using silent push notifications:
+
+1. Enable silent push:  
+  In Xcode click your **project**, select your app **target**, go to the **Signing & Capabilities** tab, add the **Background Modes** capability, then check the **Remote notifications** checkbox.
+
+2. Create an push key:  
+  Go to [https://developer.apple.com/account/resources/authkeys](https://developer.apple.com/account/resources/authkeys) and create a new APNS key. Download the key and save its Key ID.
+
+3. Upload the push key:  
+  Go to the PicUP management console, enter your app ID and key ID, and upload the key p8 file.
+
+4. Ensure push entitlement:  
+  Go to [https://developer.apple.com/account/resources/identifiers](https://developer.apple.com/account/resources/authkeys), click your app ID, and check **Push Notifications**.
+
+5. Register for remote notifications:  
+  In your `AppDelegate.application(:didFinishLaunchingWithOptions:)` call `application.registerForRemoteNotifications()`.
+
+6. Pass the push token:  
+  Save the push token you receive in `AppDelegate.application(:didRegisterForRemoteNotificationsWithDeviceToken deviceToken:)`. Alternatively, if you use Firebase Messaging, save `Messaging.apnsToken`. Then when you call `PicUpSDK.shared.register()`, pass the hex string of the token as the `uuid` argument:
+
+  ```swift
+  let tokenHexString = deviceToken.map { String(format: "%02.2hhx", $0) }.joined()
+  PicUpSDK.shared.register(name, clientPhoneNumber: number, organizationCode: orgCode, securityCode: securityCode, uuid: tokenHexString, completion: {_ in })
+  ```
+
+7. Pass the notification:  
+  In your `application(:didReceiveRemoteNotification:fetchCompletionHandler:)` call:
+
+  ```swift
+  PicUpSDK.shared.didReceiveMessage(userInfo: userInfo) { result in
```

Comment:
1. Need to add the "PicUpMpbile" check that is the message from picup
2. Add an example